libcontainer/utils/utils_unix.go

```diff
@@ -9,6 +9,9 @@ import (
 	"os"
 	"strconv"
 	"sync"
+	"syscall"
```

Comment:
According to https://github.com/opencontainers/runc/pull/3956#discussion_r1281896310 , can we switch to `x/sys/unix`?

Reply:
@utam0k I think if this one is done, this PR could be merged. Do you have a time to deal with it.

Reply:
Sure, I will be able to do it next weekend.

Reply:
@lifubang Done